DXRando/DeusEx/Classes/SmugglerElevatorTracker.uc

```diff
@@ -0,0 +1,49 @@
+class SmugglerElevatorTracker expands FlagToggleTrigger;
+
+var bool street, elevatorMoved;
+
+static function SmugglerElevatorTracker CreateSET(DXRando dxr)
+{
+    local SmugglerElevatorTracker set;
+
+    set = dxr.Spawn(class'SmugglerElevatorTracker',, 'elevatorbutton');
+    set.street = dxr.localURL == "02_NYC_STREET" || dxr.localURL == "04_NYC_STREET" || dxr.localURL == "08_NYC_STREET";
+    set.flagExpiration = dxr.dxInfo.missionNumber + 1;
+
+    return set;
+}
+
+event Tick(float deltaTime)
+{
+    if (elevatorMoved == false)
+        elevatorMoved = MoveElevator();
```

Comment:
You could disable tick after successfully moving the elevator also

Reply:
`Tick` was never enabled here in the first place. Is that something worth messing with?